src/FakeItEasy/Core/AssemblyExtensions.cs

```diff
@@ -14,16 +14,7 @@ public static bool ReferencesFakeItEasy(this Assembly assembly)
         {
             Guard.AgainstNull(assembly, nameof(assembly));
 
-#if FEATURE_REFLECTION_GETASSEMBLIES
             return assembly.GetReferencedAssemblies().Any(r => r.FullName == TypeCatalogue.FakeItEasyAssembly.FullName);
-#else
-            var fakeItEasyLibraryName = TypeCatalogue.FakeItEasyAssembly.GetName().Name;
-            var context = Microsoft.Extensions.DependencyModel.DependencyContext.Default;
-
-            var runtimeLib = context.RuntimeLibraries
-                .SingleOrDefault(library => string.Equals(library.Name, assembly.Name(), System.StringComparison.Ordinal));
-            return runtimeLib != null && runtimeLib.Dependencies.Any(dependency => string.Equals(dependency.Name, fakeItEasyLibraryName, System.StringComparison.Ordinal));

```

Comment:
Yes the API wasn't available when I started working on the project.

After the future release of NetStandard 2.0 there would be more places like this. It's probably good to have a issue to track to review this port and undo changes if they are no longer useful.